config/routes.rb

```diff
@@ -1,4 +1,5 @@
 Rails.application.routes.draw do
+  resources :pets
```

Comment:
This probably needs to go inside the `api/v1` scope so we can access it the same way we did games. Or we need to get rid of the `api/v1` scope if we don't need to access it that way, in which case we need to change the calls on the react side.

Reply:
Let's keep it and just put the `:pets` inside of the `api/v1`.